src/en/troubleshooting-logs-audit.md

```diff
@@ -0,0 +1,28 @@
+Title: Audit logging
+
+# Audit logging
+
+Juju audit logging provides a chronological account of all events by capturing
+invoked user commands. These logs reside on the controller involved in the
+transmission of commands affecting the Juju client, Juju machines, and the
+controller itself.
+
+See [Juju log][logs] documentation to learn about standard Juju logging.
+
+The audit log filename is `/var/log/juju/audit.log` and contains records which
```

Comment:
Would be useful to outline how to access/read this log.

Reply:
:ok: